yoti-sdk-impl/src/main/java/com/yoti/api/client/docs/session/create/SimpleSessionSpec.java

```diff
@@ -35,14 +36,18 @@
     @JsonProperty("required_documents")
     private final List<RequiredDocument> requiredDocuments;
 
+    @JsonProperty("block_biometric_consent")
+    private final Boolean blockBiometricConsent;
```

Comment:
Just an observation: after some discussions for the Go SDK, the conclusion was to default newly introduced `boolean` values to `false`, which might mean you could use the primitive type here?

The way you have it means it's treated as optional (omitted from payload), which I think is more consistent with the rest of the Java SDK, so I think it's fine to leave this as is.

Reply:
None of the SDKs should be providing their own default values.  Default values are decided by our _session-svc_, not the SDKs.  If the _Relying Business_ have not explicitly specified a value then the SDK should not be sending anything.

It's particularly important that this behaviour is observed for the this new _biometric_consent_ flag, as well as _manual_check_ on the authenticity check config.

Reply:
Let's leave the booleans as non-primitive types, as they are currently, to follow how the API was designed. They then can be excluded for Doc Scan API calls. We can discuss the profile service default values separately